app/models/plugins/thermometer.rb

```diff
@@ -28,4 +29,57 @@ def liquid_data
   def name
     self.class.name.demodulize
   end
+
+  def update_goal
+    self.goal = self.determine_next_goal
+    self.save
+  end
+
+  protected
+  def goal_should_update
+    current_total >= self.goal
```

Comment:
it's true it would be cool to have it say "ONE SIGNATURE NEEDED FOR 100K" and press the button. I also think it's fine either way and I'm not dying to muck around with it right now.